python/sensr_message_listener.py

```diff
@@ -19,11 +21,15 @@ def __init__(self,
                  address="localhost", 
                  listener_type=ListenerType.BOTH, 
                  output_port = "5050", 
-                 point_port = "5051"):
-        self._address = "ws://" + address
+                 point_port = "5051",
+                 crt_file_path=""):
```

Comment:
Why is this not `"../keys/sensr-sdk-ca.crt"` by default?

Reply:
At the first time I tried to set default path. But I faced a problem which the path depends on where the script was called.
So, I removed the default path and wanted to user to set the path correctly in their script.